optuna/importance/__init__.py

```diff
@@ -24,16 +25,17 @@ def get_param_importances(
     evaluator: Optional[BaseImportanceEvaluator] = None,
     params: Optional[List[str]] = None,
     target: Optional[Callable[[FrozenTrial], float]] = None,
+    normalize: bool = True,
 ) -> Dict[str, float]:
     """Evaluate parameter importances based on completed trials in the given study.
 
     The parameter importances are returned as a dictionary where the keys consist of parameter
     names and their values importances.
-    The importances are represented by floating point numbers that sum to 1.0 over the entire
-    dictionary.
-    The higher the value, the more important.
+    The importances are represented by non-negative floating point numbers, where higher values
+    mean that the parameters are more important.
     The returned dictionary is of type :class:`collections.OrderedDict` and is ordered by
     its values in a descending order.
+    If ``normalize`` is :obj:`True`, the sum of the importance values are normalized to 1.0.
```

Comment:
How about the following?
```suggestion
    By default, the sum of the importance values are normalized to 1.0.
```